src/EnergyPlus/DataOutputs.hh

```diff
@@ -94,14 +95,41 @@ namespace DataOutputs {
         std::unique_ptr<RE2> pattern;
         std::unique_ptr<RE2> case_insensitive_pattern;
     };
-    extern std::unordered_map<std::string, std::unordered_map<std::string, OutputReportingVariables>> OutputVariablesForSimulation;
+
+    // Two structs to have a case insensitive unordered_map. We need to have a case insenstive hasher and a case insensitive comparator
+    // (The default allocator for unordered_map is fine)
+    struct case_insensitive_hasher
```

Comment:
These could potentially go to `UtilityRoutines.hh` since there could be other containers that could use these. However, this isn't critical since they are fine here as well.